javascript/javascript1/week1/housey-pricey.js

```diff
@@ -0,0 +1,39 @@
+// Variables we defined for the task
+const Phouse_depth = 10;
+const Phouse_height = 10;
+const Phouse_width = 8;
+const Pgarden_size = 100;
+
+const Jhouse_depth = 11;
+const Jhouse_width = 5;
+const Jhouse_height = 8;
+const Jgarden_size = 70;
+
+// Volume calculator
+
+const volumeInMetersPeter = Phouse_height * Phouse_depth * Phouse_width;
+const volumeInMetersJulia = Jhouse_height * Jhouse_depth * Jhouse_width;
+
+// House real price calculator for Peter
+
+PhousePrice = volumeInMetersPeter * 2.5 * 1000 + Pgarden_size * 300;
+
+// Comparing real price and house cost for giving advice to Peter
+
+if (PhousePrice < 2500000) {
```

Comment:
Very minor feedback item but I would also recommend assigning the `2500000` to a variable to make it easier to understand the logic in the future. What does this 2500000 mean?